browser-api/src/main/java/com/duckduckgo/app/notification/NotificationRepository.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.notification
+
+interface NotificationRepository {
```

Comment:
Had to create this in the browser-api to be able to check in the new module if a notification already existed in the DB. I created https://app.asana.com/0/414730916066338/1201836448484376 to follow up on this and use it across the app.